fli-iam/shanoir-ng
# [Fli iam#2084] Filter processed datasets in Solr

# Changes
- Allow filtering processed dataset in Solr view
![image](https://github.com/fli-iam/shanoir-ng/assets/59697998/754de56e-d081-4e9c-b5f7-ca2781ad5e77)
- Display an icon in table indicating if processed
![image](https://github.com/fli-iam/shanoir-ng/assets/59697998/e223cedc-5f9c-489a-980d-bda8c96e8c6d)
- Change processing icon to ![image](https://github.com/fli-iam/shanoir-ng/assets/59697998/fb42e65a-509c-4443-a544-df1e8977f84f)
- Change processed dataset icon to ![image](https://github.com/fli-iam/shanoir-ng/assets/59697998/b8e190be-0889-46cb-9704-62f8128df244)

# How to test
- Import a processed dataset or execute a VIP pipeline
- Check that processed dataset is well indexed and present in Sorl table
- Check that processed datasets are filterable 
- Check that the processed icon is correctly displayed in Solr table
- Check that study tree is OK
